contents/blog/save-engineers-time.md

```diff
@@ -0,0 +1,77 @@
+
+[Feature Flags](https://posthog.com/product/feature-flags) are an example of this. Feature Flags help test new features to a subset of users, and they integrate with the rest of your product data. PostHog provides tools to non-engineers to understand Feature Flags, help set them up, and monitor their success. They have more ability to decide the "when, where, and who" of Feature Flags. For you, all you need to add is a single [if statement](https://posthog.com/manual/feature-flags#implementing-the-feature-flag) around the feature. This leaves you more time to implement the feature itself.
+
+If features fail, PostHog makes it easy to turn them off. Breaking your flow with urgent work to remove them isn't required. It just requires flipping a switch in PostHog to kill them. This is especially useful for apps that are slower to update (such as app store or extension reviews). These slow updates can take a surprising amount of time, so limiting them is critical to maximizing your useful time.
+
+As an example, the crypto wallet [Phantom](https://posthog.com/customers/phantom) uses PostHog Feature Flags as kill switches for all their features. This is because they can't deploy a new version of their Chrome Extension on demand. Feature Flags give them control of the user experience, limit feature failure, and save time.
+
+On top of Feature Flags, PostHog includes several other testing tools that are simple for you to set up and easy for non-engineers to use. They include [Session Recording](https://posthog.com/product/session-recording), [Heatmaps](https://posthog.com/product/heatmaps), [Multivariate Testing](https://posthog.com/product/experimentation-suite), and more. These create a positive feedback loop in product decisions that help features get tested faster. The team gets data on the quality of their decision and this helps them improve and speed up the next one.
+
+## Simple, customizable data ingestion
+
+When implementing Product Analytics, many services require significant work to implement and customize. They often require writing detailed requests in your app and ensuring they are handled properly in your analytics platform. PostHog makes it simple to capture events and gain data on usage. At its [simplest](https://posthog.com/docs/integrate), it requires either a one-line [snippet](https://posthog.com/docs/integrate#snippet) or an API request with the wanted data object.
+
+You don't need to spend time implementing a library for sending those API requests because we've already built them. We have [libraries](https://posthog.com/docs/integrate#libraries) in client and server languages like Javascript, Python, and Ruby. As well, we have several integrated ingestion [apps](https://posthog.com/docs/integrate#apps) like Amazon Kinesis, Shopify, and Zapier. This doesn't prevent you from implementing your own library with our API but saves you time if you need it.
```

Comment:
```suggestion
You don't need to spend time implementing a library for sending those API requests because we've already built them. We have [libraries](https://posthog.com/docs/integrate#libraries) in client and server languages such as Javascript, Python, and Ruby. As well, we have several integrated ingestion [apps](https://posthog.com/docs/integrate#apps) such as Amazon Kinesis, Shopify, and Zapier. This doesn't prevent you from implementing your own library with our API but saves you time if you need it.
```

The libraries aren't _like_ Javascript; Javascript _is_ one of the libraries. I'm that sort of editor.